src/core/Akka/Serialization/Serialization.cs

```diff
@@ -129,10 +131,9 @@ private Serializer GetSerializerByName(string name)
             {
                 if (kvp.Key.Equals(name))
                 {
-                    var serializerTypeName = kvp.Value.GetString();
-                    var serializerType = Type.GetType(serializerTypeName);
+                    if (!_serializerIdByName.TryGetValue(name, out int serializerId))
```

Comment:
I'd remove the `serializersConfig` line here - no need to read in HOCON if we're just using a cached map.

Reply:
Yeah your right. The extra check on line 132 serves no purpose.